### DISCUSSION TITLE
Scrapping Facebook

### DISCUSSION THREAD

**Discussion by FaizFaisalAhmad** [teamalgoritma/community#676] [Q&A] [answered]
bagaimana cara kita untuk scrapping Facebook

saya sudah coba berbagai cara tetep ga dapet content div nya

![image](https://github.com/teamalgoritma/community/assets/81740393/5b6188bd-bd51-4774-ac69-d8c1d746ee79)

**Comment by jafarhf1** [comment, marked as answer]
Dear @FaizFaisalAhmad,

Untuk melakukan scraping pada facebook, maka akan sulit jika hanya menggunakan library `requests` karena website tersebut sangat bergantung pada javascript. Oleh karena itu, alternatifnya Bapak bisa coba memanfaatkan library `facebook-scraper` berikut adalah dokumentasi referensinya : https://github.com/kevinzg/facebook-scraper. Alternatif lain, Bapak bisa juga menggunakan library `selenium` sebagaimana yang dijelaskan pada postingan ini : https://stackoverflow.com/a/74105681.

Demikian yang dapat kami sampaikan, semoga membantu.

Best regards,
Team Algoritma

**Reply by FaizFaisalAhmad** [reply]
ternyata susah yah untuk menscrap facebook, saya kira bisa pakai cara yang sudah dipelajari dari Capstone, :(
